docs/user/usage.rst

```diff
@@ -62,12 +67,14 @@ General syntax
    $ shaarli <global arguments> <endpoint> <endpoint arguments>
 
 
+.. note:: The following examples assume a :doc:`configuration` file is used
+
 GET info
 ~~~~~~~~
 
 .. code-block:: bash
 
-   $ shaarli -u https://host.tld/shaarli/ -s s3kr37 --output pprint get-info
+   $ shaarli --output pprint get-info
```

Comment:
Switching `output` to `format` is a good idea, writing to an output file is rather low priority since redirecting SDTIN isn't that hard.